packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts

```diff
@@ -73,11 +88,55 @@ async function setup (): Promise<void> {
   await alice.rpc.account.sendTokensToAddress({}, { [aliceColAddr]: ['10000@TSLA'] })
```

Comment:
@chee-chyuan refer LN:85 above. it is related to https://github.com/DeFiCh/jellyfish/issues/865
We need to remove code like that from tests. and take a loan instead if required.

Reply:
note that this only for loan tokens. tokens like `BTC` should be ok.